src/client/components/Review.js

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+export function Review() {
+  const { id } = useParams();
+  const [title, setTitle] = useState("");
+  const [desc, setDesc] = useState("");
+  const [star, setStar] = useState("");
+  const [filterMeal, setFilterMeal] = useState({});
+
+  console.log(filterMeal);
+  useEffect(() => {
+    fetch("/api/reviews/").then((res) => res.json()).then((data) => {
+      console.log(data);
+      setFilterMeal(data)
+    })
+  }, []);
+
+  function onSubmit() {
+    async function fetchReview() {
+      await fetch("/api/reviews", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          title: title,
```

Comment:
It could be nice to add an opportunity to add a name of person who give a review.